tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -189,9 +189,11 @@ spec:
           value: "false"
       computeResources:
         requests:
-          memory: 300Mi
+          cpu: 250m
+          memory: 500Mi
         limits:
-          memory: 300Mi
+          cpu: 2
+          memory: 2Gi
```

Comment:
This should match the request as memory is not a resource that can be easily deallocated without killing the pod.